jupyter-docker/Dockerfile

```diff
@@ -161,7 +161,7 @@ RUN apt-get install -yq --no-install-recommends python3 \
 # R Kernel
 #######################
 
-RUN apt-get -t jessie-backports install -y --no-install-recommends \
+RUN apt-get update && apt-get -t jessie-backports install -y --no-install-recommends \
```

Comment:
This didn't work on my laptop without `apt-get update`. It may have been due to my docker cache -- then again having it here shouldn't harm anything.